lms/djangoapps/verify_student/models.py

```diff
@@ -141,6 +141,34 @@ def active_at_datetime(self, deadline):
         )
 
 
+class ManualVerification(IDVerificationAttempt):
+    """
+    Each ManualVerification represents an enterprise user's verification that bypasses the need for
+    any other verification.
+    """
+
+    reason = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text=(
+            'Specifies the reason for manual verification of the user.'
+        )
+    )
+
+    class Meta(object):
+        app_label = "verify_student"
+
+    def __unicode__(self):
+        return 'ManualIDVerification for {name}, status: {status}'.format(
+            name=self.name,
+            status=self.status,
+        )
+
+    def should_display_status_to_user(self):
+        """Whether or not the status should be displayed to the user."""
```

Comment:
[Always format docstrings using the multi-line convention](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings).
```python
"""
Whether or not the manual verification status should be displayed to the user.
"""
```

Reply:
following the format previously being used in the file ...

Reply:
Lets use the standard format while making new changes.